common/sdk_util.cc

```diff
@@ -31,13 +31,6 @@ SdkUtil::SdkUtil() {
   absl::Status status = path::GetEnv("GGP_SDK_PATH", &ggp_sdk_path_env_);
   if (absl::IsNotFound(status) || ggp_sdk_path_env_.empty())
     ggp_sdk_path_env_ = path::Join(program_files_path_, "GGP SDK");
```

Comment:
I wonder if we should change this, maybe just leave it empty. Thoughts?

Reply:
My plan is to get rid of SdkUtil entirely. Right now, there are still a few usages
- GetServicesConfigPath() by AS30 for the config.json file -> Plan is to either remove it and use Absl's ability to specify a flags file, or to keep it and provide flag to specify the config file. I'll probably keep it since it's less work (still referenced by tests, for instance).
- GetLogPath() by AS30. Will provide a flag to set the log path, and log to stdout by default.
- GetDevBinPath() by LocalAssetsStreamManagerServiceImpl to find ggp.exe. This could just be removed if we rely on ggp being in PATH. Also, this code path is Stadia specific, so we could remove it or move the logic to find the full path to LocalAssetsStreamManagerServiceImpl entirely.